test/e2e/scale.go

```diff
@@ -494,11 +541,9 @@ func getClusterCreateAndWaitFn(input clusterctl.ApplyCustomClusterTemplateAndWai
 			CustomTemplateYAML:           clusterTemplateYAML,
 			ClusterName:                  clusterName,
 			Namespace:                    namespace,
-			CNIManifestPath:              input.CNIManifestPath,
 			WaitForClusterIntervals:      input.WaitForClusterIntervals,
 			WaitForControlPlaneIntervals: input.WaitForControlPlaneIntervals,
 			WaitForMachineDeployments:    input.WaitForMachineDeployments,
-			WaitForMachinePools:          input.WaitForMachinePools,
 			Args:                         input.Args,
 			PreWaitForCluster:            input.PreWaitForCluster,
```

Comment:
Maybe a rebase error?

Reply:
(see below)

I guess MPs are basically not implemented and CNIManifestPath was also not used

Reply:
Answered below - will close this comment to keep the conversation in one place - but on CNIManifestPath it's not used in this test, and doesn't really make a lot of sense to use IMO.

Reply:
MP Q answered below. The CNIManifestPath isn't used here, and doesn't really make sense with this test and the in-memory provider.